include/boost/json/parser.hpp

```diff
@@ -98,11 +101,39 @@ class parser : public basic_parser
 
     /** Default constructor.
 
+        Constructs an empty parser that uses the
+        default memory resource to allocate
+        intermediate storage.
+
+        @note
+        Before any JSON can be parsed, the function
+        @ref start must be called. 
+    */
+    BOOST_JSON_DECL
+    parser() noexcept;
+
+    /** Constructor.
+
+        Constructs a empty parser using the supplied
```

Comment:
Nitpick: I would say specified instead of supplied. But it is too small a distinction to bother changing right now.